compiler/rustc_middle/src/mir/interpret/allocation.rs

```diff
@@ -655,14 +659,16 @@ impl InitMask {
                 (u64::MAX << bita) & (u64::MAX >> (64 - bitb))
             };
             if new_state {
+                self.ensure_blocks(blocka);
                 self.blocks[blocka] |= range;
-            } else {
-                self.blocks[blocka] &= !range;
+            } else if let Some(block) = self.blocks.get_mut(blocka) {
+                *block &= !range;
```

Comment:
Please add a comment saying why it is okay to do nothing here if `get_mut` returns `None`.